### PR TITLE
Add 2 more scenarios about update alias

### PR DESCRIPTION
**Pull request by yalzhang**
Add 2 more scenarios about update alias by difference: only by mac
or only by pci address.

Signed-off-by: yalzhang <yalzhang@redhat.com>

### CODE REVIEW

**Comment by yalzhang** [comment]
depends on https://github.com/avocado-framework/avocado-vt/pull/2606

**Comment by yalzhang** [comment]
new add 2 cases, local run successfully
$ avocado run --vt-type libvirt virtual_network.iface_update.negative_test.update_alias_match_pci  virtual_network.iface_update.negative_test.update_alias_match_mac
/root/code/avocado/avocado/plugins/run.py:299: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : c2bbe4c063f7db7e6393bb488e528315f0100c31
JOB LOG    : /root/avocado/job-results/job-2020-06-23T02.08-c2bbe4c/job.log
 (1/2) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_alias_match_pci: PASS (4.98 s)
 (2/2) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.negative_test.update_alias_match_mac: PASS (5.20 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 13.58 s

File: libvirt/tests/src/virtual_network/iface_update.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ def run(test, params, env):
     del_rom = "yes" == params.get("del_rom")
     del_filter = "yes" == params.get("del_filter")
     check_libvirtd = "yes" == params.get("check_libvirtd")
+    del_mac = "yes" == params.get("del_mac")
```
Comment (chunfuwen) [inline libvirt/tests/src/virtual_network/iface_update.py:51]: del_mac = "yes" == params.get("del_mac", "no") sound better

Comment (yalzhang) [reply libvirt/tests/src/virtual_network/iface_update.py]: updated

**Review by kylazhang — APPROVED** [review]

**Pull request merged**